.github/workflows/cypress-test.yml

```diff
@@ -12,92 +12,30 @@ env:
 
 jobs:
   tests:
-    name: Run aggregation view cypress test
+    name: Run Cypress tests
     runs-on: ubuntu-latest
```

Comment:
What are your thoughts about running on Windows too? Is that something you may want to do at some point? If so I can help you with it if you need.

Reply:
Thanks for offering the help @scrawfor99! I will do the linux only for this PR, because we still need to fix some of the SAML integ test. I will let you know for sure if we add the windows as future follow up.